src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -193,10 +172,12 @@ void DisplayWindowMupen64plus::_readScreen2(void * _dest, int * _width, int * _h
 	if (_dest == nullptr)
 		return;
 
+	m_glInfo.init();
```

Comment:
ContextImpl can be initialized after we got valid GL context. But why call init in DisplayWindowMupen64plus::_readScreen2 ?